core/include/moveit/task_constructor/task.h

```diff
@@ -105,6 +105,13 @@ class Task : protected WrapperBase
 	/// remove function callback
 	void erase(TaskCallbackList::const_iterator which);
 
+	typedef std::function<void(const SolutionBase& t)> SolutionCallback;
+	typedef std::list<SolutionCallback> SolutionCallbackList;
+	/// add function to be called every time a new solution is available
+	SolutionCallbackList::const_iterator addSolutionCallback(SolutionCallback&& cb);
```

Comment:
Where's the implementation of this method?

Reply:
Well, I didn't push it...